W-12305473 helm chart config use (#230)
mulesoft/docs-gateway: gateway/1.4/modules/ROOT/pages/flex-troubleshoot-linux-services.adoc

[[other-services-failed-to-run]]
== Other Services Failed to Run

> If you error does not appear above, attempt to restart Flex Gateway:

Reviewer: ```suggestion
If your error does not appear, attempt to restart Flex Gateway:
```
